src/components/usersTable/usersTable.js

```diff
@@ -0,0 +1,37 @@
+import {Table, TableBody, TableCell, TableContainer, TableHead, TableRow} from "@material-ui/core";
+
+
+export default function UsersTable({
+  users
+}) {
+  return (
+    <TableContainer>
+      <Table sx={{minWidth: 700}} aria-label="users table">
+        <TableHead>
+          <TableRow>
+            <TableCell>Name</TableCell>
+            <TableCell alignRight>Email</TableCell>
+            <TableCell alignRight>Cell</TableCell>
+            <TableCell alignRight>Gender</TableCell>
+            <TableCell alignRight>Nationality</TableCell>
+          </TableRow>
+        </TableHead>
+        <TableBody>
+          {
+            users.map(
+              (user) => (
+                <TableRow key={Math.random().toString(36).substring(7)}>
```

Comment:
You can do a separate component for row with content and map it here

Reply:
Cool 👏. That's a good solution if we do not have any unique value in data. That's far better than leaving empty / passing index - both are the same and [can lead to rendering issues](https://reactjs.org/docs/lists-and-keys.html#keys).